app/views/layouts/application.html.erb

```diff
@@ -13,6 +13,8 @@
   <body>
     <%= render "layouts/header" %>
     <div class="container">
+      <p class="notice"><%= notice %></p>
```

Comment:
We can move these flash messages into a separate partial, like `views/shared/_flash.html.erb` then render it from here. For example in [MIA project referrence](https://github.com/nimblehq/mia-web/blob/64c275874fa085455ebb3bb4d1f7c09bf80da49f/engines/mia_auth/app/views/layouts/mia_auth/application.html.slim#L21)

Reply:
Good catch! Fixed in [`3a8b1ec` (#30)](https://github.com/suho/google-search-ruby/pull/30/commits/3a8b1ecffc4b6d94f7ad5903c24c98ec3158850b)